packages/carbon-graphs/webpack/dev-server.js

```diff
@@ -29,7 +29,7 @@ const compiler = webpack({
         test: /\.(js|jsx)$/,
         include: [
           path.join(__dirname, '../', 'src'),
-          path.join(__dirname, '../','../','../', 'dev'),
+          path.join(__dirname, '../', 'dev'),
```

Comment:
Does this fix the loading dev site in IE?

Reply:
Yes veera.